corehq/apps/app_manager/templates/app_manager/v2/module_view.html

```diff
@@ -271,9 +271,9 @@
                 </li>
               {% endif %}
             {% endfor %}
-            {% if details %}{% if  module.module_type != 'basic' or request|toggle_enabled:'COPY_CASE_CONFIGS' %}
+            {% if details %}
```

Comment:
Let's also put this behind the `advanced_app_builder` setting in v2, which is how we're planning to hide semi-advanced functionality. Similar code at https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/templates/app_manager/v2/partials/case_config.html#L76